packages/core/src/actions/smartAccount/internal/sendUserOperation.ts

```diff
@@ -40,7 +44,7 @@ export async function _sendUserOperation<
 
   const request = await client.middleware
     .signUserOperation(args.uoStruct, {
-      ...args,
+      overrides,
```

Comment:
did we need overrides here? thought the multisig signatures are in context and that's why we introduced context in addition to overrides?

Reply:
but what if someone else adds custom middleware and needs the overrides?

Reply:
also we do use the overrides to figure out how to correctly craft the aggregated signatures in the middleware

Reply:
oh, because we are not guaranteed anymore that all the middlewares are run before the internal senduseroperation. For example. `signUserOperation` action calls the signer middleware (although you removed on this PR, which I commented about below), and for that middleware, _sendUserOperation should get the overrides.